README.md

```diff
@@ -180,15 +180,13 @@ them to the appropriate format and workflow that your tool requires.
 
 
 ```bash
-pip install 'apache-airflow==2.8.2' \
- --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.8.2/constraints-3.8.txt"
+pip install 'apache-airflow==2.8.2' --constraint "https://raw.githubusercontent.com/apache/airflow/constraints-2.8.2/constraints-3.8.txt"
```

Comment:
This is working as expected. However, you might have added additional space after `\` causing the problem.

Reply:
I only know such a problem into the PowerShell. Most possible @hamad-tariq tried to install Airflow on Windows Host. Am I right? If so, I want to mention that there is no official support of Windows yet (and no plan in foreseeable future), see: https://github.com/apache/airflow/issues/10388

Reply:
yes sounds isolated. Let's leave it as is

Reply:
Yep, I agree, let's leave it as is.

I'm going to close this PR now, but thanks @hamad-tariq. I hope to see more docs PRs from you 👍

Reply:
Thanks @jedcunningham for the appreciation.

Reply:
As per my experience, I had tried the existing command multiple times with the ``` \ ``` but it only worked when I removed the ``` \ ``` . So I decided to correct it up for others so that they can directly use the given command on the Windows CLI and not run with any errors as I did. However, It'll be great if we can add both the commands with clear instructions of when to use which one of them so that if someone faces an error or problem they have the can find a solution right away with in the official documentation.

Reply:
Yes. If you can propose a description of that kind, that is a better PR.

Reply:
You just need to make sure you describe the right circumstances and explain what's going on - and you are likely the best person to do it.

Reply:
Yes I guess that's a better approach to expand the content of the documentation than replacing something in the it. Thanks to all the members for the feedback.